notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>Is "normalizing" the right word here for lightkurve - I think it does a lot more than that - maybe "lightkurve for advanced processing of our light curves" (kind of vague but probably okay at this step)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>you're right, i imported this before i used it for more than just normalizing and forgot to update the text here. Now it is fixed. </p>